fstar.opam

```diff
@@ -17,6 +17,8 @@ depends: [
   "pprint" {build}
   "sedlex" {build}
   "ppxlib" {>= "0.27.0"}
+  "ocaml-lsp-server" {>= "1.14.2" build}
```

Comment:
@tahina-pro what if I remove changes in this file. That way if somebody want VSCode it will install it locally and other files would not be affected. For VS Code any dune file needed, so if future work materialized it can be rewritten